cmake/os_flags.cmake

```diff
@@ -87,15 +87,15 @@ function(ie_avx2_optimization_flags flags)
         if(CMAKE_CXX_COMPILER_ID STREQUAL "Intel")
             set(${flags} "/QxCORE-AVX2" PARENT_SCOPE)
         elseif(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
-            set(${flags} "/arch:AVX2" PARENT_SCOPE)
+            set(${flags} "/arch:AVX /arch:AVX2" PARENT_SCOPE)
         else()
             message(WARNING "Unsupported CXX compiler ${CMAKE_CXX_COMPILER_ID}")
         endif()
     else()
         if(CMAKE_CXX_COMPILER_ID STREQUAL "Intel")
             set(${flags} "-march=core-avx2 -xCORE-AVX2 -mtune=core-avx2" PARENT_SCOPE)
```

Comment:
As I can see the Intel compiler arguments were not changed. Is it work?

Reply:
As I know in failed build (on IA32) GCC 7.5 was used. So, first of all, pay attention to the compilation options for gcc.

Reply:
@asuhov does it help to resolve some issues on your ia32 build?

Reply:
This is test PR for testing probably fixes. Review is not needed yet.